scripts/flash.py

```diff
@@ -100,41 +107,41 @@ def ihx_n_ops(ihx, addr_sector_map):
 
 # Lock sector of STM flash (0-11).
 def _stm_lock_sector(self, sector):
-  self._waiting_for_callback = True
   msg_buf = struct.pack("B", sector)
+  self.inc_n_queued_ops()
```

Comment:
Is there a way to make it less fragile than having to put inc_n_queued_ops and dec_n_queued_ops before/after the ops? Like making link.send_message do the work on n_queued_ops?